transifex/utils.py

```diff
@@ -271,53 +271,57 @@ def merge_pr(self):
 
         retries = 0
         while retries <= self.MAX_MERGE_RETRIES:
-            try:
-                self.pr.merge(merge_method=self.merge_method)
-                logger.info('Merged [%s/#%d].', self.name, self.pr.number)
-                return True
-            except github.GithubException as e:
-                # Assumes only one commit is present on the PR.
-                statuses = self.pr.get_commits()[0].get_statuses()
-
-                # Check for any failing Travis builds. If any are found, notify the team
-                # and move on.
-                if any('travis' in s.context and s.state == 'failure' for s in statuses):
+            # Assumes only one commit is present on the PR.
+            statuses = self.pr.get_commits()[0].get_statuses()
+            for s in statuses:
+                logger.info('[%s] status has status [%s]', s, s.state)
+            if any(s.state == 'failure' for s in statuses):
```

Comment:
Do we still need to check for `'travis in s.context'`, as in https://github.com/edx/ecommerce-scripts/pull/78/files#diff-5f7e2de26d943a8474540224f60a064aL284 ?

Reply:
during my debugging of edx-platform job none of the job context had travis in it. also that could be changed without our knowledge by devops, so removed that condition. now it checks for the status of all tests running and when the final state is success it merges.